packages/components/src/disabled/types.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import type { ComponentProps } from 'react';
+
+export interface DisabledProps extends ComponentProps< 'div' > {
```

Comment:
Since we're using ges/components/src/ui/context/wordpress-component.ts#L13-L24, there's no need to extend `ComponentProps` here

```suggestion
export interface DisabledProps {
```